gravityflow/gravityflow
# Fix missing extension icons under Forms Toolbar Settings

## Description
This fixes the missing extension icons which are displayed under the Forms Toolbar Settings.

This resolves [gravityflow/backlog#126](https://github.com/gravityflow/backlog/issues/126).

## Testing instructions
1. Create a simple form.
2. Hover on the Toolbar Settings in the form
3. Note many Gravity Flow Extension icons are missing (blank).
4. Switch to this branch.
5. Repeat Step 2 and confirm the icons now display fine.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes
N/A

## Checklist:
